tests/apps/pubsub-publisher/app.go

```diff
@@ -37,9 +38,17 @@ const (
 	appPort      = 3000
 	daprPortHTTP = 3500
 	daprPortGRPC = 50001
-	pubsubName   = "messagebus"
+	PubSubEnvVar = "DAPR_TEST_PUBSUB_NAME"
```

Comment:
Why are we defining this env. var? It isn't used by the current test. Is there any expected use for this?

Reply:
This file is not the test itself it is an app to help the e2e test to execute,

The "pubsubName" variable was already defined as a const, I changed to var

This var serves to use the same app image but change the messagebus name that should be subscribed/published so that, when running the e2e test a can do that by changing that envVar otherwise I can't test against a different pluggable component.